semgrep/tests/qa/test_public_repos.py

```diff
@@ -60,7 +60,12 @@ def xfail_repo(url, *, reason=None):
         ),
     ],
 )
-def test_semgrep_on_repo(run_semgrep_in_tmp, repo_url):
+def test_semgrep_on_repo(monkeypatch, tmp_path, repo_url):
+    TESTS_PATH = Path(__file__).parent.parent
```

Comment:
Not really sure what's going on with all this path stuff - I trust @underyx's thoughts :+1:

Reply:
it's just replicating the code from `run_semgrep_in_tmp` fixture, we good 😇